core/templates/layout.user.php

```diff
@@ -60,7 +60,11 @@
 			<div class="header-left">
 				<a href="<?php print_unescaped($_['logoUrl'] ?: link_to('', 'index.php')); ?>"
 					id="nextcloud">
-					<div class="logo logo-icon"></div>
+					<div class="logo logo-icon">
+						<span class="hidden-visually">
+							<?php p($l->t('%s\'s homepage', [$theme->getName()])); ?>
```

Comment:
```suggestion
							<?php p($l->t('%s homepage', [$theme->getName()])); ?>
```
Was the suggestion.
Also otherwise you might get `...s's homepage` which is not good language